src/core/sock.c

```diff
@@ -438,6 +456,16 @@ int nn_sock_getopt_inner (struct nn_sock *self, int level,
             strncpy (optval, self->socket_name, *optvallen);
             *optvallen = strlen(self->socket_name);
             return 0;
+        case NN_SEC_ATTR:
+            memcpy (optval, self->sec_attr, self->sec_attr_size);
```

Comment:
I'm not sure I understand the style rule here.. we have strncpy with a space after it above, but strlen without.  Do functions get a space between the name and open paren, or not?  (I prefer without.  But I prefer consistency even more.)